pages/co-workingSpace/details.tsx

```diff
@@ -0,0 +1,168 @@
+
+              Features of this coworking space are Private Offices, Meeting rooms, and event space. As at the time of this research to rank coworking spaces in Africa, they ranked the highest in Nairobi, Kenya.
+            </p>
+            <div className="bg-light_green lg:w-[30%] md:w-[30%] p-2 my-10 ">
+              <h3 className="font-bold text-[16px] text-center text-sunglow">AVAILABLE SPACE</h3>
+            </div>
+            <div className="xlg:w-full lg:w-[95%] md:w-full mt-4 mb-8">
+              <div className="grid md:grid-cols-5 md:grid-rows-1 grid-rows-3 md:border md:border-gray-300 rounded xlg:text-[11px] md:text-[12px] text-[13px] text-primary_grey font-bold">
+                {availableSpaceTabsData.map((tab) => (
+                  <Tabs key={tab.id} id={tab.id} title={tab.title} />
+                ))}
+              </div>
+            </div>
+            <div className="my-10 xlg:w-full lg:w-[95%] md:w-full w-[95%] lg:mx-0 sm:m-auto">
+              <WorkSpaceTable
+                availableSpaces={availableSpacesTableData}
+                TableWrapper="border-separate border-spacing-1"
+                headerCellClass="bg-sunglow  text-white text-center "
+                cellClassName="py-4 bg-grey_2 text-center"
+                buttonWrapper=" py-4 bg-white border-primary_green "
+              />
+            </div>
+          </div>
+
+          <div className="lg:col-span-2 ">
+            <div className="my-10 ">
+              <div className="bg-light_green rounded-t-lg pl-5 py-5">
+                <h1 className="text-[24px] text-[600] leading-[150%]text-dark_black">Business Hours</h1>
+              </div>
+              {businessHoursData.map((hours) => (
+                <div className={`grid grid-cols-4 py-6 bg-white_2  ${hours.day === "Sunday" ? "border-b-0 rounded-b-lg shadow-0" : "border-border_b border-b-2"
+                  }`} key={hours.id}>
+                  <div className="text-grey_3 xlg:text-[13px] md:text-[16px] text-[10px] text-[400] mt-2 pl-5">{hours.day} :</div>
+                  {hours.startTime && hours.endTime ? (
+                    <div className="col-span-2 flex mt-2 xlg:text-[13px] md:text-[16px] text-[10px] text-[400] justify-between px-5 text-grey_3"><span className="">{hours.startTime} </span> - <span className="">{hours.endTime}</span>
+                    </div>
+                  ) : (
+                      <div className="col-span-2 xlg:text-[13px] md:text-[16px] text-[10px] text-[400] flex mt-2 justify-start px-5 text-grey_3"><span className="">Closed</span>
+                    </div>
+                  )}
+                  <div className="">
+                    <Button
```

Comment:
can you make the reserve now button active?
 a modal is supposed to show on the click of the button.
 here is a link to the pop up design 
 https://www.figma.com/file/VxoknphZySezGNvfr63r42/Afrisplash-Final-UI?type=design&node-id=4590-52884&mode=design&t=IEncb3KeUJdpzkIp-0

Reply:
I will implement this and make a push, it was an oversight on my part.

Reply:
Hello Sammy, This has been implemented.